app/main/views/briefs.py

```diff
@@ -194,6 +195,90 @@ def update_brief_status(brief_id):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/<any(publish, withdraw):action>', methods=['PUT'])
+def update_brief_status_by_action(brief_id, action):
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.framework.status != 'live':
+        abort(400, "Framework is not live")
+
+    action_to_status = {
+        'publish': 'live',
+        'withdraw': 'withdrawn'
+    }
+    if brief.status != action_to_status[action]:
+        brief.status = action_to_status[action]
+
+        validate_brief_data(brief, enforce_required=True)
+
+        audit = AuditEvent(
+            audit_type=AuditTypes.update_brief_status,
+            user=updater_json['updated_by'],
+            data={
+                'briefId': brief.id,
+                'briefStatus': brief.status,
+            },
+            db_object=brief,
+        )
+
+        db.session.add(brief)
+        db.session.add(audit)
+        db.session.commit()
+
+    return jsonify(briefs=brief.serialize()), 200
+
+
+@main.route('/briefs/<int:brief_id>/create-draft-from-withdrawn-brief', methods=['POST'])
+def create_draft_brief_from_withdrawn_brief(brief_id):
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if brief.framework.status != 'live':
+        abort(400, "Framework is not live")
+
+    if brief.status != 'withdrawn':
+        abort(400, "Brief status is not 'withdrawn'")
+
+    new_brief_data = brief.data
+    new_brief_data['title'] = 'Copy of ' + brief.data['title']
+    new_brief = Brief(
+        data=new_brief_data,
+        framework=brief.framework,
+        lot=brief.lot,
+        users=brief.users
+    )
+    validate_brief_data(new_brief, enforce_required=True)
```

Comment:
The new brief is a draft and we didn't accept any additional data (apart from the title change we've applied), so we're mostly revalidating the original brief again. I don't think we have to do this.

The draft brief will have to go through validation when published, so the only difference is whether we fail to make a copy or the created draft has validation issues (which should be fine for missing fields, but could be a bit trickier for invalid values like overflowing title as we don't show a very useful error message when the validation fails on a field that doesn't belong to the current form)